src/parsers/gas.js

```diff
@@ -108,3 +116,13 @@ const getTxFee = (gasPrice, gasLimit, priceUnit, nativeCurrency) => {
     },
   };
 };
+
+export const gweiToWei = gweiAmount => {
```

Comment:
When it comes to gas related operations, we should stick with using bignumbers just to future-proof ourselves. Can do this in a later PR